### DISCUSSION TITLE
[Feature Request?] PDF Notes for Marp-VSCode users

### DISCUSSION THREAD

**Discussion by zhangchuck** [marp-team/marp#149] [Ideas] [answered]
@yhatt Not sure if you are already working on this, but are there any plans to add the `--pdf-notes` functionality from Marp-CLI (https://github.com/marp-team/marp-cli/pull/369) to Marp-for-VSCode? As discussed in the previous issues (https://github.com/marp-team/marp-cli/issues/261), if you are presenting using PDFs, this is really helpful beacuse it allows for easy integration with PDF presentation software like pympress (and also [pdfpc](https://github.com/pdfpc/pdfpc/pull/293).

Perhaps it could be set using a directive in the YAML front-matter or as an "export setting" of some sort?

**Comment by yhatt** [comment]
Yes, I'll work on it this weekend. 

> * I'm going to add new preference about PDF metadata and `author`/ `keywords` completion support in another pull requests.
>
> _-- https://github.com/marp-team/marp-vscode/pull/275_

"PDF metadata" actually means `--pdf-notes` option support.

**Comment by yhatt** [comment, marked as answer]
Shipped `markdown.marp.pdf.noteAnnotations` setting with the latest [Marp for VS Code v1.3.0](https://github.com/marp-team/marp-vscode/releases/tag/v1.3.0).

![](https://user-images.githubusercontent.com/3993388/130329452-16787638-239d-450b-9ef1-113c6d62e31f.png)

**Reply by zhangchuck** [reply]
Thanks! Just tested it out. This works great.
